README.md

```diff
@@ -61,7 +61,7 @@ The following table lists the configurable parameters of the `kubernetes-externa
 
 | Parameter                            | Description                                                  | Default                                                 |
 | ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------- |
-| `env.AWS_DEFAULT_REGION`             | Set AWS_DEFAULT_REGION in Deployment Pod                     | `us-west-2`                                             |
```

Comment:
we got rid of this already with the last merged PR https://github.com/godaddy/kubernetes-external-secrets/commit/05b92a2aa5f648f9269bbf8f8a16cffa6bb5f998#diff-04c6e90faac2675aa89e2176d2eec7d8L64. Please sync your fork with our latest master.

Reply:
![image](https://user-images.githubusercontent.com/6058569/57643824-a8d45300-7588-11e9-888f-412410c88646.png)

Reply:
Will do, what happens if you don't declare a variable for `AWS_REGION`? Because you have removed this entirely in master.